api/auth/user_management.py

```
@@ -234,11 +234,26 @@ async def validate_user(request: Request) -> Tuple[Optional[Dict[str, Any]], boo
+        # If still not found, also accept Authorization: Bearer <token>
+        if not api_token:
+            auth_header = (
+                request.headers.get("authorization")
+                or request.headers.get("Authorization")
+            )
+            if auth_header:
+                try:
+                    parts = auth_header.split(None, 1)
+                    if len(parts) == 2 and parts[0].lower() == "bearer":
+                        api_token = parts[1].strip()
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)